app/components/domain-form.tsx

```diff
@@ -0,0 +1,51 @@
+import { AddIcon, InfoIcon } from '@chakra-ui/icons';
+import { Button, HStack, Input, Select, Textarea, Tooltip, VStack } from '@chakra-ui/react';
+import { Form } from '@remix-run/react';
+
+export default function DomainForm() {
+  return (
+    <Form className="domain-form" method="post">
+      <VStack maxW="xl" spacing="5">
+        <HStack w="100%">
+          <Input placeholder="Domain Name" name="name" />
+          <Tooltip>
+            <InfoIcon></InfoIcon>
```

Comment:
These need text to be useful.

Reply:
I am not sure what text to display here. Do we have something or should I add text based on what I understand? @Myrfion @Ririio

Reply:
You can add the text based on your understanding, and we can take a look at it and see if it's correct